src/api.rs

```diff
@@ -84,13 +84,14 @@ pub struct EncoderConfig {
   pub min_quantizer: u8,
   pub bitrate: i32,
   pub tune: Tune,
-  /// log2(tile columns). Unused if tiles is specified.
-  pub tile_cols_log2: usize,
-  /// log2(tile rows). Unused if tiles is specified.
-  pub tile_rows_log2: usize,
-  /// Number of tiles desired. The video is split automatically so that
-  /// it contains at least this many tiles. Overrides tile_cols_log2
-  /// and tile_rows_log2.
+  /// Number of tiles horizontally. Encoder may round this number up
+  /// due to bitstream limitations.
```

Comment:
up or down. For example, if user passes `--tile-rows 6`, it will convert internally to use `tile_rows_log2 = 3`.

If the input is 600px, there will be 5 tiles.
If the input is 700px, there will be 6 tiles.
If the input is 800px, there will be 7 tiles.

To avoid confusion, maybe we should force `--tile-rows` and `--tile-cols` to accept only powers of 2?

Reply:
ehhhhh yeah. Though technically it can still go up with >4k video.

Reply:
Updated it to reject non powers of 2.